backend/src/ml_space_lambda/training_job/lambda_functions.py

```diff
@@ -36,6 +42,7 @@
 
 @api_wrapper
 def create(event, context):
+    deleted_metric_definitions = False
```

Comment:
Do we need this when we assign it a value on line 107?

Reply:
This is being set to true in two different places and then assigns the value once. We could just remove this and then assign the value twice.